assets/styles/app.scss

```
@@ -14,3 +14,34 @@ label.required > .fr-x-label-content::after {
 .fr-x-icon--xl::before {
     --icon-size: 2.5rem;
 }
+
+.fr-x-text--right {
+    text-align: right;
+}
+
+.fr-x-text--center {
+  text-align: center;
+}
+
+.landing-jumbotron__illustration {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+h2 {
+    color: var(--blue-france-sun-113-625);
+}
+
+h1 {
+    color: var(--blue-france-sun-113-625);
+}
+
+section.how-it-works {
+    background-color: var(--blue-france-975-75);
+
+    h2 {
+        color: var(--grey-50-1000);
+    }
+}
```

Ça applique ce style à tous les h1/h2, ça ne me semble pas désirable

Si on ne souhaite pas importer le fichier de couleurs en l'absence de tree-shaking, on peut au moins définir manuellement la classe du DSFR et l'utiliser là où on en a besoin

Il n'y aura alors pas besoin de forcer `how-it-works > h2` à reprendre la couleur standard

```suggestion
.fr-title--blue-france {
    color: var(--blue-france-sun-113-625);
}

.fr-background-alt--blue-france {
    color: var(--blue-france-975-75);
}
```

---

Done